compiler/rustc_target/src/spec/base/linux_musl.rs

```diff
@@ -8,8 +8,5 @@ pub(crate) fn opts() -> TargetOptions {
     base.post_link_objects_self_contained = crt_objects::post_musl_self_contained();
     base.link_self_contained = LinkSelfContainedDefault::InferredForMusl;
 
-    // These targets statically link libc by default
-    base.crt_static_default = true;
-
```

Comment:
Question: do we have any way to test (or rather, track) the "finalized" crt static for each target (as in, after the defaults and overrides are merged together to produce the final value)? So that if anyone *does* modify the default and cause the "finalized" value to change, we do so *intentionally* with something like a test rebless or test modification?

I looked at `--print=all-target-specs-json` locally, but AFAICT that omits this field if it's not overriden or something...

Reply:
Yeah, I added a run-make test that captures the current state.